### PR TITLE
Cleaned up the App-Bar

### PR DESCRIPTION
**Pull request by RahulMishra0722**
### PR Fixes:
- 1 Made the app bar cleaner by adding a Toggle Options

Resolves #[Issue Number if there] 

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue
Screenshot 2024-04-16 at 11 08 10 AM
Screenshot 2024-04-16 at 11 03 07 AM

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,18 +0,0 @@
-NEXT_PUBLIC_BASE_URL_LOCAL=http://127.0.0.1:3000
-ADMIN_SECRET="ADMIN_SECRET"
-JWT_SECRET="JWT_SECRET"
-# DONT CHANGE FOR RUNNING WITH DOCKER
```
Comment (hkirat) [inline .env.example:4]: why did u get rid of this?

File: package.json [hunk 2]
```diff
@@ -20,14 +20,14 @@
...
-    "studio:docker" : "open http://localhost:5555 || start http://localhost:5555"
+    "studio:docker": "open http://localhost:5555 || start http://localhost:5555"
   },
   "dependencies": {
     "@auth/prisma-adapter": "^1.0.6",
     "@discordjs/core": "^1.1.1",
     "@discordjs/next": "^0.1.1-dev.1673526225-a580768.0",
     "@icons-pack/react-simple-icons": "^9.4.0",
-    "@prisma/client": "^5.6.0",
+    "@prisma/client": "^5.12.1",
```
Comment (hkirat) [inline package.json:30]: lets push this to another PR

**Review by hkirat — CHANGES_REQUESTED** [review]
Please get rid of pnpm lock file

**Comment by nimit9** [comment]
Hey this has been fixed in #309

File: .env.example (hunk 1 above)
Comment (RahulMishra0722) [reply .env.example:4]: No i didn't

**Comment by RahulMishra0722** [comment]
> Please get rid of pnpm lock file

Ok on it

**Comment by RahulMishra0722** [comment]
> Please get rid of pnpm lock file
here is the fixed version
https://github.com/code100x/cms/pull/446#issue-2247005279

**Pull request closed without merging**